### DISCUSSION TITLE
Window changes place when toggle max layout

### DISCUSSION THREAD

**Discussion by gruvw** [qtile/qtile#3852] [Q&A]
Using the following shortcut twice `Key([mod], "m", lazy.window.toggle_maximize(), desc="Toggle maximize")` on the right window in the column layout with two windows opened swaps them...

How to prevent the modification of the order of windows when toggling max layout ?

**Comment by ervinpopescu** [comment]
This is a thing I've been experiencing too, no idea if it's intended behavior.
